content/300-guides/500-other/880-troubleshooting-orm/100-help-articles/700-netlify-caching-issue.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: 'Netlify build dependency caching workaround'
+metaTitle: 'Learn to configure your build process on Netlify to avoid caching-related problems'
+metaDescription: 'Learn to configure your build process on Netlify to avoid caching-related problems'
+---
+
+## Problem
+
+If you deploy an application using Prisma to [Netlify](https://netlify.com/), you may run into the following error message on deployment:
+
+```
+Prisma has detected that this project was built on Netlify, which caches dependencies. This leads to an outdated Prisma Client because Prisma's auto-generation isn't triggered. To fix this, make sure to run the \`prisma generate\` command during the build process.
+
+Learn how: https://pris.ly/d/netlify-build
+```
```

Comment:
See changes above for Vercel

Reply:
Since this is part of the error message, I think we can leave the URL here.